behaviour_and_protocol/values.md

```diff
@@ -1,23 +1,16 @@
-# Abtion values
+# Collective ownership
+Collective ownership gives control of the entire project to all team members. "Your part" is the product. Either it's a good product, or it is not. 
 
-## Solve real problems
-Practices:
-  - Focus on client value
-  - Simplicity
+## Collective vision
+- Open communication: No privileged information.
+- Cut the middle person. We put the team directly in front of the stakeholder.
+- Break knowledge silos. No one is "the expert" in an area.
 
-## Long-term-value focus
-Practices:
-  - TDD
-  - Quality code base
-  - Knowledge sharing
+## Collective accountability
+- Work end-to-end. We need to be able to impact as much of the project as possible.
+- T-shape profiles. We still have roles, and they know about areas surrounding their expertise.
```

Comment:
It seems like the adjective "T-shaped" is more common:
```suggestion
- T-shaped profiles. We still have roles, and they know about areas surrounding their expertise.
```

Reply:
Could this be something like:
`T-shaped profiles. We each have strengths, but we understand areas outside of our expertise.`

Reply:
I feel that this one needs another turn. I feel "strengths" is a too vague word. 
I'm going to keep it out of this PR, but feel free to open a new one or just go ahead, pair up with someone and improve the wording.